ci-operator/step-registry/aws/provision/iam-user/minimal-permission/aws-provision-iam-user-minimal-permission-commands.sh

```diff
@@ -246,6 +314,11 @@ servicequotas:ListAWSDefaultServiceQuotas
 tag:GetResources
 EOF
 
+if [[ ${CREDENTIALS_MODE} != "Manual" ]]; then
```

Comment:
Sounds only `mint` mode need the follow permission, but `passthrough` does not need them, right?